nekoyume/Assets/_Scripts/UI/Widget/LobbyMenu.cs

```diff
@@ -724,6 +731,16 @@ public override void Show(bool ignoreShowAnimation = false)
             UpdateButtons();
             stakingLevelIcon.sprite =
                 stakeIconData.GetIcon(States.Instance.StakingLevel, IconType.Bubble);
+
+            var thorSchedule = Nekoyume.Game.LiveAsset.LiveAssetManager.instance.ThorSchedule;
+            if (thorSchedule is null || !thorSchedule.IsOpened)
+            {
+                thorSeasonPassButton.gameObject.SetActive(false);
+            }
+            else
+            {
+                thorSeasonPassButton.gameObject.SetActive(true);
+            }
```

Comment:
```suggestion
            thorSeasonPassButton.gameObject.SetActive(thorSchedule?.IsOpened);
```
체크는 안해봤는데 이렇게 간략화 시킬 수 있지 않을까요?

Reply:
5abd0786dec7d9521efaa57644a2cd72e969158d 에서 간단하게 해봤습니다